python/.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
I never use pre-commit-config but when lint it, it affects all the folders, not only python.

Reply:
Oh wait, that's right, I forgot to run pre-commit. Imma fix it!